symfony/routing/5.1/config/packages/routing.yaml

```diff
@@ -0,0 +1,11 @@
+framework:
+    router:
+        utf8: true
+
+        # Replace these node to configure the "request context" to use to generate URLs
+        # in non-HTTP contexts, such as CLI commands.
+        # See https://symfony.com/doc/current/routing.html#generating-urls-in-commands
```

Comment:
This doc is not up-to-date yet, but there is probably no reason to move it elsewhere.